docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -58,23 +58,27 @@ For a description of parameters, see the [CREATE query description](../../../sql
 
 -   `ENGINE` — Name and parameters of the engine. `ENGINE = MergeTree()`. The `MergeTree` engine does not have parameters.
 
--   `PARTITION BY` — The [partitioning key](custom-partitioning-key.md).
-
-    For partitioning by month, use the `toYYYYMM(date_column)` expression, where `date_column` is a column with a date of the type [Date](../../../sql-reference/data-types/date.md). The partition names here have the `"YYYYMM"` format.
-
 -   `ORDER BY` — The sorting key.
 
     A tuple of columns or arbitrary expressions. Example: `ORDER BY (CounterID, EventDate)`.
 
--   `PRIMARY KEY` — The primary key if it [differs from the sorting key](#choosing-a-primary-key-that-differs-from-the-sorting-key).
+    Use the `ORDER BY tuple()` syntax, if you don't need sorting. In this case, ClickHouse stores data in the order of inserting. If you want to save data order when inserting data by `INSERT ... SELECT` queries, set [max_insert_threads = 1](../../../operations/settings/settings.md#settings-max-insert-threads).
```

Comment:
Actually it's not so important detail that makes this text much less readable (it's rarely used beyond ClickHouse tests). We'd better put it into a separate article on how to properly choose a sorting key for MergeTree tables (maybe in F.A.Q or here in `mergetreee-family` folder).